contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -152,27 +146,33 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
     }
 
     bytes32 workflowExecutionId;
```

Comment:
You could save gas and number of vars by returning `workflow_report_id` (see proposal https://docs.google.com/document/d/1VlosqVXzJhWP77ss4UOH0b4rFX2J4sgfdzt0-prHHw0/edit#heading=h.crycehnu4byi), because workflow execution ID is never used directly.